Portal/src/Datahub.Portal/Pages/Home.razor

```diff
@@ -9,7 +9,7 @@
 @inject ILocalStorageService _localStorage
 @inject IDialogService _dialogService
 
-@page "/home/{TabIndex?}"
+@page "/home"
```

Comment:
The TabIndex was to allow history in the homepage I think that was history from NRCAN, I assume that's why we don't need it.

Reply:
Yeah, I removed the tab component from the homepage. I didn't delete the file in case we want to bring it back but I feel like there are better UX options available